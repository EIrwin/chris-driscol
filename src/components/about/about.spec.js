// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import About from './about.component';

const mockAbout = {
  tagLine: 'tag line',
  imageUrl: 'https://fake/image',
  imageTitle: 'image title',
  imageCaption: 'image caption',
  description: ['description 1', 'description 2'],
};

it('renders correctly', () => {
  const tree = renderer.create(<About about={mockAbout} />).toJSON();
  expect(tree).toMatchSnapshot();
});
