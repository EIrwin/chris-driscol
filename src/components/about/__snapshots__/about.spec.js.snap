// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="home-section aboutme"
  id="aboutme"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-lg-12 text-center"
      >
        <h2
          className="section-heading"
        >
          About Me
        </h2>
        <p
          className="section-subheading text-muted"
        >
          This should help you get to know more about me..
        </p>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-sm-8"
      >
        <p
          className="large"
          dangerouslySetInnerHTML={
            Object {
              "__html": "description 1",
            }
          }
        />
        <p
          className="large"
          dangerouslySetInnerHTML={
            Object {
              "__html": "description 2",
            }
          }
        />
      </div>
      <div
        className="col-sm-4"
      >
        <div
          className="me"
        >
          <img
            alt="image title"
            className="img-responsive img-circle"
            src="https://fake/image"
          />
          <h1>
            image title
          </h1>
          <h2
            className="text-muted"
          >
            image caption
          </h2>
        </div>
      </div>
      <div
        className="col-lg-8 col-lg-offset-2 text-center about-tag"
      >
        <p
          className="large text-muted"
        >
          I 
          <svg
            fill="currentColor"
            height="1em"
            preserveAspectRatio="xMidYMid meet"
            style={
              Object {
                "color": undefined,
                "verticalAlign": "middle",
              }
            }
            viewBox="0 0 40 40"
            width="1em"
          >
            <g>
              <path
                d="m20 37.1q-0.6 0-1-0.4l-13.9-13.4q-0.2-0.2-0.6-0.6t-1.3-1.4-1.5-2.2-1.2-2.7-0.5-3.1q0-4.9 2.8-7.7t7.9-2.7q1.4 0 2.8 0.4t2.7 1.3 2.1 1.6 1.7 1.5q0.8-0.8 1.7-1.5t2.1-1.6 2.7-1.3 2.8-0.4q5 0 7.9 2.7t2.8 7.7q0 4.9-5.1 10l-13.9 13.4q-0.4 0.4-1 0.4z"
              />
            </g>
          </svg>
           working on 
          <strong>
            Agile teams
          </strong>
           motivated by 
          <strong>
            delivering customer value
          </strong>
           early and often.
        </p>
      </div>
    </div>
  </div>
</section>
`;
